cosmo_tester/test_suites/bootstrap_based_tests/sanity_scenario_test.py

```diff
@@ -0,0 +1,174 @@
+########
+# Copyright (c) 2017 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#    * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    * See the License for the specific language governing permissions and
+#    * limitations under the License.
+
+import time
+import pytest
+
+from cosmo_tester.framework.examples.nodecellar import NodeCellarExample
+from cosmo_tester.framework.test_hosts import TestHosts
+from cosmo_tester.framework.util import prepare_and_get_test_tenant
+
+from cosmo_tester.test_suites.snapshots import (
+    create_snapshot,
+    download_snapshot,
+    upload_snapshot,
+    restore_snapshot,
+    # upgrade_agents,    # bug in agents upgrade
```

Comment:
Put the Jira number here, so we'll know what to do when we fix it

Reply:
fixed